frontend/src/scenes/insights/InsightTooltip/InsightTooltip.scss

```diff
@@ -0,0 +1,89 @@
+@import '~/vars';
+
+.ph-graph-tooltip {
+    padding: 0 !important;
+    overflow: hidden;
+    border: 1px solid $border;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
+    font-size: 14px;
+    z-index: $z_graph_tooltip;
+    transition: all 0.4s;
+    max-width: 480px;
+    border-radius: $radius;
+    background: #fafafa;
+
+    .LemonTable {
+        font-size: 0.75rem;
+        border: none;
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+        border-bottom: 1px solid $border;
+
+        .color-column {
+            padding: 0;
+            width: 6px;
+        }
+
+        .datum-column {
+            padding-left: 0.5rem;
+        }
+
+        .LemonTable__scroll {
+            width: 100%;
+            overflow: hidden;
+        }
+        .LemonTable__content > table > thead {
+            text-transform: none;
+
+            .datum-column {
+                .LemonTable__header-content {
+                    white-space: nowrap;
+                }
+            }
+            .series-column-header {
+                margin-left: $default_spacing / 4;
+            }
+        }
+
+        .LemonTable__content > table > tbody {
+            .color-column {
+                position: relative;
+
+                .color-cell {
+                    position: absolute;
+                    top: 0;
+                    bottom: 0;
+                    width: 6px;
+                }
+            }
+            .datum-label-column {
+                font-weight: 600;
+            }
+            .series-data-cell {
+                font-weight: 600;
+            }
+            .tag-pill {
+                background-color: rgba(0, 0, 0, 0.08);
+                margin-right: 0;
+                border: 0;
+                color: $bg_navy;
+                overflow: hidden;
+                vertical-align: middle;
+            }
+        }
+    }
+
+    .table-subtext {
+        font-size: 0.75rem;
+        text-align: center;
+        padding: $default_spacing / 2 $default_spacing;
+        color: $text_muted;
+        background-color: $bg_light;
```

Comment:
Nit: let's minimize usage of preprocessor variables. See https://github.com/PostHog/posthog/pull/7726#discussion_r770674509.
There's also IMO an additional problem with `$default_spacing` in particular, in that it's quite opaque what it is, using it requires additional math like the division here AND there is little utility in making it a variable (it's not like we're going to EVER change it across the board – in reality the spacing must be tuned everywhere anyway). The `rem` unit (which is `16px` in PostHog too) seems like a better way of specifying sizes.

Reply:
thanks, thats good to know!